src/main/java/lesson11Task1/NumbersCalculator.java

```diff
@@ -0,0 +1,32 @@
+package lesson11Task1;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class NumbersCalculator {
+    public static void main(String[] args) {
+        calculateNumbersBy3(List.of(3, 6, 9, 10, 15, 17, 18, 20, 21));
+
+    }
+
+    public static boolean calculateNumbersBy3(List<Integer> numbers) {
```

Comment:
function should return a list of numbers that are divisible by 3. You are returning boolean value here